types/README.md

```diff
@@ -0,0 +1,35 @@
+# Workers Types Generator
+
+This directory contains scripts for automatically generating TypeScript types
+from [JSG RTTI](../src/workerd/jsg/rtti.h).
+
```

Comment:
Nit: not blocking for this PR but a request for later: It would be worthwhile documenting the dependencies used here... specifically the minimum Node.js version. I know the bazel stuff brings in the right version but just for detail information, especially given that the tool uses a number of experimental Node.js APIs that are fairly recent additions.